File: Fragments2/app/src/main/java/com/example/fragments/MainActivity.java
package com.example.fragments;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Log.i("CICLOVIDA","ActivityOnCreate");

        Toast.makeText(this, "Mensaje desde Activity", Toast.LENGTH_SHORT).show();
    }
}

File: PruebasDebugger/app/src/main/java/es/ua/eps/pruebasdebugger/MainActivity.kt
package es.ua.eps.pruebasdebugger

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    companion object{
        private const val TAG = "MiActivity"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        Log.i(TAG, "indice=idx")
        Toast.makeText(this,"HOLAAA", Toast.LENGTH_LONG).show();
        Toast.makeText(this,"HOLAAA2", Toast.LENGTH_LONG).show();
        Toast.makeText(this,"HOLAAA3", Toast.LENGTH_LONG).show();
        Toast.makeText(this,"HOLAAA4", Toast.LENGTH_LONG).show();

    }
}